File: app/src/main/java/com/danielspeixoto/connect/module/SignUp.java
package com.danielspeixoto.connect.module;


import com.danielspeixoto.connect.model.pojo.User;

/**
 * Created by danielspeixoto on 2/14/17.
 */

public class SignUp {

    public interface View extends ActivityBase.View {

    }

    public interface Presenter extends ActivityBase.Presenter {
        void signUp(User user);
    }
}

File: app/src/main/java/com/danielspeixoto/connect/module/SignUp.kt
package com.danielspeixoto.connect.module

import com.danielspeixoto.connect.model.pojo.User


/**
 * Created by danielspeixoto on 2/14/17.
 */

class SignUp {

    interface View : ActivityBase.View

    interface Presenter : ActivityBase.Presenter {
        fun signUp(user: User)
    }
}
